app/main.ipynb

```diff
@@ -126,19 +158,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "selected_elements = array[[1,  4, 7]]\n",
+    "\n",
+    "result = selected_elements.sum()\n",
+    "print(result)"
```

Comment:
The indices used for selecting elements from the array do not sum up to 27. You need to adjust the indices to select the correct elements that sum to 27.